config/config.go

```diff
@@ -16,17 +16,7 @@ package config
 
 import (
 	"github.com/coreos/ignition/v2/config/shared/errors"
-	"github.com/coreos/ignition/v2/config/util"
-	"github.com/coreos/ignition/v2/config/v3_0"
-	types_3_0 "github.com/coreos/ignition/v2/config/v3_0/types"
-	"github.com/coreos/ignition/v2/config/v3_1"
-	trans_3_1 "github.com/coreos/ignition/v2/config/v3_1/translate"
-	types_3_1 "github.com/coreos/ignition/v2/config/v3_1/types"
-	"github.com/coreos/ignition/v2/config/v3_2"
-	trans_3_2 "github.com/coreos/ignition/v2/config/v3_2/translate"
-	types_3_2 "github.com/coreos/ignition/v2/config/v3_2/types"
 	"github.com/coreos/ignition/v2/config/v3_3_experimental"
```

Comment:
Similarly, if we import this as something like `exp`, there's one less thing to change at stabilization time.